cluster-infra/jaeger-operator/kustomization.yaml

```diff
@@ -6,7 +6,8 @@ namespace: observability
 
 resources:
   - ../namespace
-  - https://github.com/jaegertracing/jaeger-operator/releases/download/v1.57.0/jaeger-operator.yaml
+  - localized.yaml
```

Comment:
For all these things, we should have an upstream file that points upstream, then a localize should be ran in workflows, and likely an edit. Basically what the monorepo does. That way we can still get dep updates.

Reply:
@adrielp, it's a little different with these resource references in the kustomization yaml. The monorepo is localizing kustomizations from a remote repository with the `kustomize localize` command. That only works with remote git repositories that have a kustomization yaml. 

Whereas what you have today are just download links to a standard manifest file.